app/src/main/java/net/squanchy/speaker/view/SpeakersView.java

```diff
@@ -0,0 +1,83 @@
+package net.squanchy.speaker.view;
+
+import android.content.Context;
+import android.support.annotation.Nullable;
+import android.support.v7.util.DiffUtil;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.AttributeSet;
+
+import java.util.List;
+
+import net.squanchy.speaker.Speaker;
+
+public class SpeakersView extends RecyclerView {
+
+    private SpeakerAdapter adapter;
+
+    public SpeakersView(Context context) {
```

Comment:
This is the last remaining bit.
Please remove this constructor and we are ready to go